docs/docs/10.4-test-utils.md

```diff
@@ -11,7 +11,10 @@ next: clone-with-props.html
 ### Simulate
 
 ```javascript
-Simulate.{eventName}(DOMElement element, object eventData)
+Simulate.{eventName}(
+  DOMElement element,
+  [object eventData]
```

Comment:
I like this particular change, with the brackets. Makes it much more clear that it's an object "flavor"